crates/runtime/src/runtime.rs

```diff
@@ -103,30 +91,6 @@ impl RuntimeBuilder {
         }
     }
 
-    /// Set a [`KeyValueAtomics`] handler to use for all component instances unless overriden for the instance
```

Comment:
Why are these removed?

Reply:
I commented out while getting to compile - I just need to think about the best way to architect this with parameters being passed as bytes... as encoding and decoding depend on store context, I need to find a way to somehow ove the context through to the handler, to then generate as these, all pass the bytes to these handlers.

I added some comments here:
Temporarily commented out some trait implementations for capability providers to allow for compilation. These need to be implemented back in before the end of this pull request to support first-hand interfaces in wasmCloud @ricochet . Need to implement with newest WRPC API, so these will likely change in form anyway.
To resolve this, we need to determine the best way to map different invocation types. Options include:

Finding a way to fulfill these requirements now that wRPC does not generate the functions for serving directly.
Continuing to use wrpc_interface_http, wrpc_interface_blobstore, etc., crate dependencies in parallel.

We can carry on directly supporting the crates from before I believe?

Reply:
I don't think we should need to change anything, the "bytes" approach should only be used for the dynamic use cases (e.g. `call`), for static bindings, `wasmcloud_host` should decode and encode typed values, like it does now